### PR TITLE
Add 404 exception to deployment if ethernet does not exists

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/deployment/views.py [hunk 1]
```diff
@@ -86,6 +87,9 @@ def ipxe(request, deployment_id=None):
             deployment = Deployment.objects.get(id=deployment_id)
         else:
             deployment = Deployment.get_deployment_for_ip(ip)
+    except Ethernet.DoesNotExist:
+        logger.warning('Ethernet does not exists for ip: {}'.format(ip))
```
Comment (mkurek) [inline src/ralph/deployment/views.py]: Ethernet -> Deployment

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**